external/netstandard/netstandard.depproj

```diff
@@ -55,8 +55,16 @@
     <PropertyGroup>
       <_NETStandardRefFolder>$(PackagesDir)$(NETStandardLibraryPackageId.ToLower())\$(NETStandardLibraryPackageVersion)\build\$(_NETStandardTFMFolder)\ref</_NETStandardRefFolder>
     </PropertyGroup>
+
+    <ItemGroup>
+      <ExcludeNetStandardRefs Include="System.ComponentModel.Composition" />
+      <NetStandardRefs
+        Include="$(_NETStandardRefFolder)\*.dll"
+        Exclude="@(ExcludeNetStandardRefs -> '$(_NETStandardRefFolder)\%(Identity).dll')" />
```

Comment:
Wes couldn't you have just done 
```xml
<Reference Include="@(NetStandardRefs)" Condition="'%(Identity)' != 'System.ComponentModel.Composition'">
```

Reply:
Yes but that doesn't scale as nicely if we need to add more to the list.